src/pybind/mgr/pg_autoscaler/module.py

```diff
@@ -210,6 +214,60 @@ def shutdown(self):
         self.log.info('Stopping pg_autoscaler')
         self._shutdown.set()
 
+    def check_pool_activity(check_time, pool_stats, pool_id, final_pg_target, old_values):
+        total = (int(pool_stats.get("rd", 0)) + int(pool_stats.get("wr", 0)))
+        if pool_id in old_values.keys():
+            if check_time == 1:
+                if old_values[pool_id]['times'] == total:
+                    old_values[pool_id]['unused'] += 1
+                    old_values[pool_id]['add'] = 0
+                    old_values[pool_id]['actual_add'] = 0
+                    if final_pg_target - old_values[pool_id]['unused'] < PG_NUM_MIN:
+                        old_values[pool_id]['unused'] -= 1
+                    else:
+                        for k1 in old_values:
```

Comment:
See comment below, can we move the conditional outside the for loop?

Reply:
change total to total_request and 'times' to 'total_requests'